packages/flutter/test/material/range_slider_test.dart

```diff
@@ -1660,7 +1660,7 @@ void main() {
       sliderTheme: const SliderThemeData(
         valueIndicatorColor: Color(0xff000001),
         overlappingShapeStrokeColor: Color(0xff000002),
-        showValueIndicator: ShowValueIndicator.always,
```

Comment:
Can you leave these tests with .always in place to ensure it does not break during deprecation?